mat/list_test.go

```diff
@@ -197,9 +203,16 @@ func legalTypesVecDenseVecDense(a, b Matrix) bool {
 	return true
 }
 
-// legalTypesNotVecVec returns whether the first input is an arbitrary Matrix
+// legalTypesNotVectorVector returns whether the first input is an arbitrary Matrix
+// and the second input is a Vector.
+func legalTypesNotVectorVector(a, b Matrix) bool {
```

Comment:
How about renaming to legalTypesMatrixVector?

Reply:
Thanks, I was looking for something like that.